10.0/Dockerfile

```diff
@@ -47,6 +47,8 @@ COPY ./templates /templates
 COPY ./before-migrate-entrypoint.d /before-migrate-entrypoint.d
 COPY ./start-entrypoint.d /start-entrypoint.d
 COPY ./MANIFEST.in /odoo
+# Place coveragerc in workdir where coverage will be launched from
+COPY ./.coveragerc /
```

Comment:
so common files are taken automatically I guess?

Reply:
thanks for the pointer :)

Reply:
The `cp` in `build.sh` must be changed to `cp -r common/{.*,*} ${TMP}` to include the hidden files

Reply:
good catch!